news/2020-06-26-mappers-report-May-June.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: "The Mappers' Report: May/June"
+date: 2020-06-30 12:00:00 +0000
+---
+
+Have you missed us? I bet you did! Sorry for the wait but here we are again, the fourth and fifth issue of the Mappers' Report in one post! Full with news and happenings all over the mapping community in osu! Did your favorite mapper won a contest? Or got the long awaited ranking criteria change finally happen? — Jump right in and find out!
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+This month's issue of the Mappers' Report contains following topics:
+
+- **[Ranking Criteria Changes](#rc-changes)**
+- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
+- **[Mappers' Showcase](#mappers-showcase)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Around the Community](#around-community)**
+- **[Conclusion](#conclusion)**
+
+## <a id="rc-changes"></a>Ranking Criteria Changes
+
+As every month we keep you up to date with what happened in the Ranking Criteria. All rule and guideline changes are listed below and in addition you can see open proposals to give your feedback to them!
+
+### All Modes
+
+- **[Remove glossaries from RC pages](https://osu.ppy.sh/community/forums/topics/1064466)**: The content of the glossaries will be put into own wiki pages! This happened already for the general Ranking Criteria and the osu! Ranking Criteria. The other will follow soon!
+- **[Allow .ogg files as a main audio source](https://osu.ppy.sh/community/forums/topics/1021547)**: Always struggle with your map getting uploaded because of too large file size? Then we have good news! You may now use .ogg as audio to save some file size! Note that this still must follow the regular kbit/s rules!
```

Comment:
```suggestion
- **[Allow .ogg files as a main audio source](https://osu.ppy.sh/community/forums/topics/1021547)**: Always struggle with your map getting uploaded because of too large file size? Then we have good news! You may now use .ogg for the song audio to save some file size! Note that this still must follow the regular kbit/s rules!
```